cilium/cmd/bpf_policy_get.go

```diff
@@ -34,26 +37,53 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var printIDs bool
+var (
+	printIDs bool
+	allList  bool
+)
+
+var policyMapRegExp = regexp.MustCompile("cilium_policy_[0-9]*$|cilium_policy_reserved_[0-9]*$")
 
 // bpfPolicyListCmd represents the bpf_policy_list command
 var bpfPolicyListCmd = &cobra.Command{
-	Use:    "get",
-	Short:  "List contents of a policy BPF map",
-	PreRun: requireEndpointID,
+	Use:   "get",
+	Short: "List contents of a policy BPF map",
 	Run: func(cmd *cobra.Command, args []string) {
 		common.RequireRootPrivilege("cilium bpf policy get")
-		listMap(cmd, args)
+		if allList {
+			listAllMaps()
+			return
+		}
+		requireEndpointID(cmd, args)
+		listMap(args)
 	},
 }
 
 func init() {
 	bpfPolicyCmd.AddCommand(bpfPolicyListCmd)
 	bpfPolicyListCmd.Flags().BoolVarP(&printIDs, "numeric", "n", false, "Do not resolve IDs")
+	bpfPolicyListCmd.Flags().BoolVarP(&allList, "all", "", false, "Dump all policy maps")
 	command.AddJSONOutput(bpfPolicyListCmd)
 }
 
-func listMap(cmd *cobra.Command, args []string) {
+func listAllMaps() {
+	mapRootPrefixPath := bpf.MapPrefixPath()
+	files, err := ioutil.ReadDir(mapRootPrefixPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	for _, file := range files {
+		if policyMapRegExp.MatchString(file.Name()) {
```

Comment:
https://golang.org/pkg/path/filepath/#Glob

Reply:
@aanm nice! Changed to use this. Let me know what you think.